README.md

```diff
@@ -209,9 +209,208 @@ public void cvurl() {
         System.out.println("CVurl POST: " + result);
     }
 ```
+And another example - due to underlying Mapper (by default it's Jackson) You can send use Java objects as a Request body without a hassle
+```java
+ public UserCreatedResponseDto createUser(UserDto userDto) {
+        return cVurl.POST(HOST + USERS)
+                .body(userDto)
+                .header(HttpHeader.CONTENT_TYPE, MIMEType.APPLICATION_JSON)
+                .build()
+                .asObject(UserCreatedResponseDto.class, HttpStatus.CREATED);
```

Comment:
this example is not correct - method asObject(class, statusCode) returns optional.

Reply:
Thx for noticing, my fault